tests/sweep_framework/sweeps/eltwise/unary_complex/is_imag.py

```diff
@@ -0,0 +1,123 @@
+        + gen_shapes([1, 1], [256, 256], [1, 1], 16),
+        "input_a_dtype": [ttnn.bfloat16],
+        "input_a_layout": [ttnn.TILE_LAYOUT],
+        "input_a_memory_config": [ttnn.DRAM_MEMORY_CONFIG, ttnn.L1_MEMORY_CONFIG],
+        "output_memory_config": [ttnn.DRAM_MEMORY_CONFIG, ttnn.L1_MEMORY_CONFIG],
+    },
+    "xfail": {
+        "input_shape": gen_shapes([1, 1, 1, 1], [6, 12, 256, 256], [1, 1, 1, 1], 16)
+        + gen_shapes([1, 1, 1], [12, 256, 256], [1, 1, 1], 16)
+        + gen_shapes([1, 1], [256, 256], [1, 1], 16),
+        "input_a_dtype": [ttnn.bfloat16, ttnn.bfloat8_b],
+        "input_a_layout": [ttnn.TILE_LAYOUT],
+        "input_a_memory_config": [ttnn.DRAM_MEMORY_CONFIG, ttnn.L1_MEMORY_CONFIG],
+        "output_memory_config": [ttnn.DRAM_MEMORY_CONFIG, ttnn.L1_MEMORY_CONFIG],
+    },
+}
+
+
+# This is the run instructions for the test, defined by the developer.
+# The run function must take the above-defined parameters as inputs.
+# The runner will call this run function with each test vector, and the returned results from this function will be stored.
+# If you defined a device_mesh_fixture above, the object you yielded will be passed into this function as 'device'. Otherwise, it will be the default ttnn device opened by the infra.
+def run(
+    input_shape,
+    input_a_dtype,
+    input_a_layout,
+    input_a_memory_config,
+    output_memory_config,
+    *,
+    device,
+) -> list:
+    data_seed = random.randint(0, 20000000)
+    torch.manual_seed(data_seed)
+
+    torch_input_tensor_ar = gen_func_with_cast_tt(
+        partial(torch_random, low=0.01, high=100, dtype=torch.float32), input_a_dtype
+    )(input_shape)
+
+    torch_input_tensor_ac = gen_func_with_cast_tt(
+        partial(torch_random, low=0.01, high=100, dtype=torch.float32), input_a_dtype
+    )(input_shape)
+
+    torch_output_tensor = torch.isreal(
+        torch.complex(torch_input_tensor_ar.to(torch.float32), torch_input_tensor_ac.to(torch.float32))
+    )
+
+    input_tensor_ar = ttnn.from_torch(
+        torch_input_tensor_ar,
+        dtype=input_a_dtype,
+        layout=input_a_layout,
+        device=device,
+        memory_config=input_a_memory_config,
+    )
+
+    input_tensor_ac = ttnn.from_torch(
+        torch_input_tensor_ac,
+        dtype=input_a_dtype,
+        layout=input_a_layout,
+        device=device,
+        memory_config=input_a_memory_config,
+    )
+    input_tensor_a = ttnn.complex_tensor(input_tensor_ar, input_tensor_ac)
+
+    start_time = start_measuring_time()
+    result = ttnn.is_imag(input_tensor_a, memory_config=output_memory_config)
+    output_tensor = ttnn.to_torch(result)
+
+    e2e_perf = stop_measuring_time(start_time)
+
+    pcc = check_with_pcc(torch_output_tensor, output_tensor, 0.99)
+    # print(f"pcc {pcc} input_a_dtype {input_a_dtype}")
+    return [pcc, e2e_perf]
+
+
+# from tests.sweep_framework.permutations import *
+
+# start_time = start_measuring_time()
```

Comment:
What are these comments here?

Reply:
This part of code was used for debugging earlier, not required at all and I deleted it.